package.json

```diff
@@ -12,24 +12,24 @@
     "README.md"
   ],
   "scripts": {
-    "build": "npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack --env.DEV=no",
+    "build": "npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack --env DEV=no",
     "build:e2e": "webpack --config ./tests/e2e/server/webpack.config.js",
-    "clean": "shx rm -rf bundles docs lib lib-esm",
-    "clean:all": "npm run clean && shx rm -rf node_modules package-lock.json && git clean -fX -e '!*.pem'",
-    "lint": "eslint --ignore-path .gitignore '**/*.ts' --fix",
+    "clean": "shx rm -rf ./bundles ./docs ./lib ./lib-esm",
+    "clean:all": "npm run clean && shx rm -rf ./node_modules ./package-lock.json && git clean -fX -e \"!*.pem\"",
+    "lint": "eslint --ignore-path .gitignore \"./**/*.ts\" --fix",
```

Comment:
Windows seems to be quirky when it comes to single quotes with globs and files or paths without a leading `./`.